allennlp/requirements.txt

```diff
@@ -1,2 +1,5 @@
 allennlp>=2.2.0
+# This constraint is necessary to use the old Pydantic.
+# See https://github.com/pydantic/pydantic/issues/5821.
+typing_extensions<4.6.0
```

Comment:
[Notes] This entry guided to install `typing_extensions==4.5.0`, and the CI jobs passed.

https://github.com/optuna/optuna-examples/actions/runs/5066098278/jobs/9095467241